src/Core/Grid/Definition/Factory/TitleGridDefinitionFactory.php

```diff
@@ -0,0 +1,234 @@
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Grid\Definition\Factory;
+
+use PrestaShop\PrestaShop\Core\Form\ChoiceProvider\GenderChoiceProvider;
+use PrestaShop\PrestaShop\Core\Grid\Action\Bulk\BulkActionCollection;
+use PrestaShop\PrestaShop\Core\Grid\Action\GridActionCollection;
+use PrestaShop\PrestaShop\Core\Grid\Action\Row\RowActionCollection;
+use PrestaShop\PrestaShop\Core\Grid\Action\Row\Type\LinkRowAction;
+use PrestaShop\PrestaShop\Core\Grid\Action\Type\SimpleGridAction;
+use PrestaShop\PrestaShop\Core\Grid\Column\ColumnCollection;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\ActionColumn;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\BulkActionColumn;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\Common\ImageColumn;
+use PrestaShop\PrestaShop\Core\Grid\Column\Type\DataColumn;
+use PrestaShop\PrestaShop\Core\Grid\Filter\Filter;
+use PrestaShop\PrestaShop\Core\Grid\Filter\FilterCollection;
+use PrestaShop\PrestaShop\Core\Hook\HookDispatcherInterface;
+use PrestaShopBundle\Form\Admin\Type\SearchAndResetType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\NumberType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Class TitleGridDefinitionFactory creates definition for title grid.
+ */
+final class TitleGridDefinitionFactory extends AbstractGridDefinitionFactory
+{
+    use BulkDeleteActionTrait;
+    use DeleteActionTrait;
+
+    public const GRID_ID = 'title';
+
+    private $genderChoiceProvider;
+
+    public function __construct(HookDispatcherInterface $hookDispatcher = null, GenderChoiceProvider $genderChoiceProvider)
+    {
+        parent::__construct($hookDispatcher);
+        $this->genderChoiceProvider = $genderChoiceProvider;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getId()
+    {
+        return self::GRID_ID;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getName()
+    {
+        return $this->trans('Titles', [], 'Admin.Navigation.Menu');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getColumns()
+    {
+        return (new ColumnCollection())
+            ->add(
+                (new BulkActionColumn('title_bulk'))
+                    ->setOptions([
+                        'bulk_field' => 'id_gender',
+                    ])
+            )
+            ->add(
+                (new DataColumn('id_gender'))
+                    ->setName($this->trans('ID', [], 'Admin.Global'))
+                    ->setOptions([
+                        'field' => 'id_gender',
+                    ])
+            )
+            ->add(
+                (new DataColumn('name'))
+                    ->setName($this->trans('Social title', [], 'Admin.Shopparameters.Feature'))
```

Comment:
```suggestion
                    ->setName($this->trans('Title', [], 'Admin.Global'))
```
Maybe we could use "Title" only here to be consistent with the rest of the page.